README.md

```diff
@@ -93,145 +60,127 @@ After you're finished please remove all the comments and instructions!
   </ul>
 </details>
 
-<!-- Features -->
 
-### Key Features <a name="key-features"></a>
 
-> Describe between 1-3 key features of the application.
+### Key Features <a name="key-features"></a>
 
-- **[key_feature_1]**
-- **[key_feature_2]**
-- **[key_feature_3]**
+- Book Motorcycles
+- View Motorcycle Info
+- View Motorcycles by category
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- LIVE DEMO -->
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-> Add a link to your deployed project.
-
-- [Live Demo Link](https://yourdeployedapplicationlink.com)
+- In progress
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- GETTING STARTED -->
+
+
 
 ## 💻 Getting Started <a name="getting-started"></a>
 
-> Describe how a new developer could make use of your project.
+> To get a local copy up and running, follow these steps.
 
-To get a local copy up and running, follow these steps.
+Clone this repository to your desired folder:
 
-### Prerequisites
+```bash
+$ git clone https://github.com/naanmohammed/Book-Motorcycles-Backend.git
+$ cd Book-Motorcycles-Backend
+```
 
-In order to run this project you need:
 
-<!--
-Example command:
 
-```sh
- gem install rails
-```
- -->
+### Prerequisites
 
-### Setup
+In order to work on this project, you need to have the following dependencies installed:
 
-Clone this repository to your desired folder:
+- [Ruby](https://www.ruby-lang.org/en/)
+- [Postgresql](https://www.postgresql.org/)
+- [Node.js](https://nodejs.org/en/)
+- [Rails](https://rubyonrails.org/)
 
-<!--
-Example commands:
 
-```sh
-  cd my-folder
-  git clone git@github.com:myaccount/my-project.git
-```
---->
 
 ### Install
 
 Install this project with:
 
-<!--
-Example command:
-
-```sh
-  cd my-project
-  gem install
+```bash
+$ bundle install
 ```
---->
+- Database Setup: Navigate to the project directory and set up the database.
+    - rails db:create
+     -rails db:migrate
+To get all the gems required for the project
 
-### Usage
+Then, run
 
-To run the project, execute the following command:
+```bash
+$ npm install
+```
+
+To install packages such as style linters
 
-<!--
-Example command:
+To check linters locally, use
 
-```sh
-  rails server
 ```
---->
+bash
+$ rubocop
+$ npx stylelint "**/*.{css,scss}"
+```
 
-### Run tests
 
-To run tests, run the following command:
 
-<!--
-Example command:
+### Usage
 
-```sh
-  bin/rails test test/models/article_test.rb
+To run the project, execute the following command:
+
+```
+rails server
 ```
---->
 
-### Deployment
 
-You can deploy this project using:
 
-<!--
-Example:
+### Run tests
 
-```sh
+To run tests, run the following command:
 
-```
- -->
+  ```sh
+    rspec ./spec
```

Comment:
- [ ]  You are using Rspec, but the "test" folder still exists, which is used by another test framework. Kindly consider deleting that folder if you are not using it. 👍